docs/zh/use_case/dolphinscheduler_task_demo.md

```diff
@@ -48,50 +48,61 @@ dpkg-reconfigure dash
 ```
 输入`no`。
 
-**运行 OpenMLDB集群与 Predict Server**
+**源数据准备**
+
+下文使用的工作流会从`/tmp/train_sample.csv`导入数据到OpenMLDB，所以，先将源数据下载到这一地址：
+```
+curl -SLo /tmp/train_sample.csv https://openmldb.ai/download/dolphinschduler-task/train_sample.csv
+```
+
+**运行 OpenMLDB 集群与 Predict Server**
 
 在容器中运行以下命令启动 OpenMLDB cluster：
 ```
 /work/init.sh
 ```
 
-我们将完成一个导入数据，离线训练，训练成功后模型上线的工作流。模型上线的部分，可以使用`/work/talkingdata`中的的predict server来完成。将它运行至后台：
+我们将完成一个导入数据，离线训练，训练成功后模型上线的工作流。模型上线的部分，将发送模型地址给predict server来完成。所以，我们先下载并后台运行predict server：
+```
+cd /work
+curl -SLo predict_server.py https://openmldb.ai/download/dolphinschduler-task/predict_server.py
+python3 predict_server.py --no-init > predict.log 2>&1 &
 ```
-python3 /work/talkingdata/predict_server.py --no-init > predict.log 2>&1 &
+```{tip}
+如果在“在线预测测试”中返回了错误，请查看日志`/work/predict.log`。
 ```
 
-**运行 DolphinScheduler**
+**下载并运行 DolphinScheduler**
+
+DolphinScheduler 支持 OpenMLDB Task 的版本，我们直接提供了一个可供下载版本，下载地址[dolphinscheduler-bin](http://openmldb.ai/download/dolphinschduler-task/apache-dolphinscheduler-dev-SNAPSHOT-bin.tar.gz)。
```

Comment:
does it has an official release?

Reply:
It's better to wait 3.1.3, the current version 3.1.2 can't run OpenMLDB Task